app/lib/server/functions/removeUserFromRoom.js

```diff
@@ -31,6 +43,16 @@ export const removeUserFromRoom = function(rid, user, options = {}) {
 		Meteor.defer(function() {
 			// TODO: CACHE: maybe a queue?
 			callbacks.run('afterLeaveRoom', user, room);
+
+			try {
+				Promise.await(Apps.triggerEvent(AppEvents.IPostRoomUserLeave, room, user));
+			} catch (error) {
+				if (error instanceof AppsEngineException) {
+					throw new Meteor.Error('error-app-prevented', error.message);
+				}
+
+				throw error;
+			}
```

Comment:
The "post" event doesn't need the `try...catch` clause, because it can't _really_ prevent that event from happening. Errors here can be suppressed for now.

Reply:
Ok! I'll fix that, thanks for poiting it out.